pycam/jpeg.py

```diff
@@ -234,10 +234,10 @@ def camera_get_picture_jpg():
 		ser.write(ack)
 
 		received = ser.read(512)
-		size = received[4]
-		temp = received[3]
-		temp *= 256
-		size += temp
+		#size = received[4]
```

Comment:
Just delete thing you don't need. If you want it again that's why we have git commit history :)